ortools/linear_solver/xpress_interface.cc

```diff
@@ -1912,16 +1912,10 @@ MPSolver::ResultStatus XpressInterface::Solve(MPSolverParameters const& param) {
 
   int xpress_stat = 0;
   if (mMip) {
-    if (this->maximize_)
-      status = XPRSmaxim(mLp, "g");
-    else
-      status = XPRSminim(mLp, "g");
+    status = XPRSmipoptimize(mLp, "g");
```

Comment:
Flag "g" is not supported by `XPRSmipoptimize` and `XPRSlpoptimize`.

It means
> g the MIP problem will be solved, calling [XPRSmipoptimize](https://www.fico.com/fico-xpress-optimization/docs/latest/solver/optimizer/HTML/XPRSmipoptimize.html) ( [MIPOPTIMIZE](https://www.fico.com/fico-xpress-optimization/docs/latest/solver/optimizer/HTML/XPRSmipoptimize.html)).

It seems that no flag should be given.

See
https://www.fico.com/fico-xpress-optimization/docs/latest/solver/optimizer/HTML/XPRSmaxim.html
https://www.fico.com/fico-xpress-optimization/docs/latest/solver/optimizer/HTML/XPRSlpoptimize.html
https://www.fico.com/fico-xpress-optimization/docs/latest/solver/optimizer/HTML/XPRSmipoptimize.html

Reply:
I agree with @flomnes , keep the flag empty (""), hoping that XPRSmipoptimize would use the DEFAULTALG to solve the initial relaxation (not explicited in documentation unfortunately)

Reply:
done